docs/UsersGuide/source/Tutorial.rst

```diff
@@ -0,0 +1,589 @@
+
+.. hint:: 
+   
+   Later, users may want to conduct additional experiments using the FV3_HRRR and FV3_WoFS_v0 physics suites. Like FV3_RRFS_v1beta, these physics suites were designed for use with high-resolution grids for storm-scale predictions. 
+
+Next, users will need to modify the data parameters in ``task_get_extrn_ics:`` and ``task_get_extrn_lbcs:`` to use HRRR and RAP data rather than FV3GFS data. Users will need to change the following lines in each section:
+
+.. code-block:: console
+
+   task_get_extrn_ics:
+     EXTRN_MDL_NAME_ICS: HRRR
+     EXTRN_MDL_SOURCE_BASEDIR_ICS: /path/to/UFS_SRW_App/develop/input_model_data/HRRR/${yyyymmddhh}
+   task_get_extrn_lbcs:
+     EXTRN_MDL_NAME_LBCS: RAP
+     EXTRN_MDL_SOURCE_BASEDIR_LBCS: /path/to/UFS_SRW_App/develop/input_model_data/RAP/${yyyymmddhh}
+     EXTRN_MDL_LBCS_OFFSET_HRS: '-0'
+
+HRRR and RAP data are better than FV3GFS data for use with the FV3_RRFS_v1beta physics scheme because these datasets use the same physics :term:`parameterizations` that are in the FV3_RRFS_v1beta suite. They focus on small-scale weather phenomena involved in storm development, so forecasts tend to be more accurate when HRRR/RAP data are paired with FV3_RRFS_v1beta and a high-resolution (e.g., 3-km) grid. Using HRRR/RAP data with FV3_RRFS_v1beta also limits the "spin-up adjustment" that takes place when initializing with model data coming from different physics.
+
+``EXTRN_MDL_LBCS_OFFSET_HRS:`` This variable allows users to use lateral boundary conditions (LBCs) from a previous forecast run that was started earlier than the start time of the current forecast configured in this experiment. This variable is set to 0 by default except when using RAP data; with RAP data, the default value is 3, so the forecast will look for LBCs from a forecast started 3 hours earlier (i.e., at 2019061515 --- 15z --- instead of 2019061518). To avoid this, users must set ``EXTRN_MDL_LBCS_OFFSET_HRS`` explicitly. 
+
+Add a section to ``config.yaml`` to increase the maximum wall time (``WTIME_RUN_POST``) for the postprocessing tasks. The wall time is the maximum length of time a task is allowed to run. On some systems, the default of 15 minutes may be enough, but on others (e.g., NOAA Cloud), the post-processing time exceeds 15 minutes, so the tasks fail. 
+
+.. code-block:: console
+
+   task_run_post:
+     WTIME_RUN_POST: 00:20:00
+
+Lastly, users must set the ``COMOUT_REF`` variable in the ``task_plot_allvars:`` section to create difference plots that compare output from the two experiments. ``COMOUT_REF`` is a template variable, so it references other workflow variables within it (see :numref:`Section %s <TemplateVars>` for details on template variables). ``COMOUT_REF`` should provide the path to the ``control`` experiment forecast output using single quotes as shown below:
```

Comment:
I searched the current documentation for what `COMOUT` is short for or what meaning it's trying to convey ("common output directory"?  If so, what's "common" about it?) but didn't see anything.  In the past, I've tried to find this out from operations people at EMC but failed.  Have you been able to find out?  It would be great to have this (and other NCO variables) documented so users can form a better mental image of the NCO directory structure and the logic behind it.  Just last week I was talking to a user and they told me it was very confusing -- not just this variable but other operational ones like NET -- because their nomenclature/purpose is not explained well (and he was an experienced modeler but not from the operations side).

Reply:
Unfortunately, I haven't been able to find out... About the best I've got is a reference to the NCO Implementation Standards in the ConfigWorkflow chapter where we define all the relevant NCO-specific variables (https://srw-ug.readthedocs.io/en/text-tutorial/ConfigWorkflow.html#nco-specific-variables). I just added the same reference where I talk about COMOUT_REF in case users come across that first. I agree that it would be useful to know/explain more, but it seems like we've both hit a bit of a brick wall on this one... Part of me wonders if the issue is that the logic behind the naming is lost to the annals of history (i.e., operations folks just use it without necessarily knowing/wondering where the names came from originally). Will update if I find anything more though!

Reply:
@gspetro-NOAA Yeah I think you're right about it being lost to the annals of history.  In the `plot_all_vars` task, it seems like we can rename this variable to something more straightforward like `POST_OUTPUT_DIR` because as far as I can tell, this task isn't something that EMC/NCO runs and so doesn't require NCO-compliant names like `COMOUT_...`.  Would that require a separate PR?

Reply:
@gsketefian I think it would be a separate PR, although I like the suggestion. That said, I believe the name was selected by @danielabdi-noaa, and he may have had reasons for selecting that name.